### PR TITLE
Adding layout for non-latin choises to target

### PR DESCRIPTION
**Pull request by Chocobo-ts**
Additional US layout and toggle hotkey for choosed non-latin layouts in target system

### CODE REVIEW

File: usr/lib/live-installer/installer.py [hunk 1]
```diff
@@ -416,9 +416,17 @@ def finish_install(self, setup):
             if(line.startswith("XKBMODEL=")):
                 newconsolefh.write("XKBMODEL=\"%s\"\n" % setup.keyboard_model)
             elif(line.startswith("XKBLAYOUT=")):
+                if setup.keyboard_layout in ('am', 'ara', 'ben', 'bd', 'bg', 'bt', 'by', 'deva', 'ge',
+                      'gh', 'gr', 'guj', 'guru', 'il', 'in', 'ir', 'iku',
+                      'kan', 'kh', 'kz', 'la', 'lao', 'lk', 'mk', 'mm', 'mn',
+                      'mv', 'mal', 'ori', 'pk', 'ru', 'scc', 'sy', 'syr',
+                      'tel', 'th', 'tj', 'tam', 'ua', 'uz'):
```
Comment (kernc) [inline usr/lib/live-installer/installer.py]: Define the list somewhere separately:

```py
NONLATIN_KEYBOARD_LAYOUTS = ...
```
Will also make it defined in a single place and can be reused (e.g. as in https://github.com/linuxmint/live-installer/pull/89).

File: usr/lib/live-installer/installer.py [hunk 2]
```diff
@@ -416,9 +416,17 @@ def finish_install(self, setup):
...
+                    setup.keyboard_layout = 'us,%s' % self.setup.keyboard_layout
```
Comment (kernc) [inline usr/lib/live-installer/installer.py]: But why add the US layout?

File: usr/lib/live-installer/installer.py [hunk 3]
```diff
@@ -416,9 +416,17 @@ def finish_install(self, setup):
...
                 newconsolefh.write("XKBLAYOUT=\"%s\"\n" % setup.keyboard_layout)
             elif(line.startswith("XKBVARIANT=") and setup.keyboard_variant is not None):
                 newconsolefh.write("XKBVARIANT=\"%s\"\n" % setup.keyboard_variant)
+            elif(line.startswith("XKBOPTIONS=")):
+                newconsolefh.write("XKBOPTIONS=grp:alt_shift_toggle")
```
Comment (kernc) [inline usr/lib/live-installer/installer.py]: Isn't this a little to much to assume and pre-set for the user? Is <kbd>Alt</kbd>+<kbd>Shift</kbd> standard shortcut to switch layouts?

Comment (Chocobo-ts) [reply usr/lib/live-installer/installer.py]: @kernc  - https://github.com/linuxmint/lmde-3-cinnamon-beta/issues/10
Yes, Alt+Shift is most frequently shortcut for it
And everything in this fix - based on ubiquity behavoir

File: usr/lib/live-installer/installer.py (hunk 2 above)
Comment (Chocobo-ts) [reply usr/lib/live-installer/installer.py]: @kernc  - https://github.com/linuxmint/lmde-3-cinnamon-beta/issues/10

**Pull request merged**